core-bundle/tests/Security/Authentication/Token/TokenCheckerTest.php

```diff
@@ -368,4 +399,9 @@ private function mockSessionWithToken(TokenInterface $token): SessionInterface
 
         return $session;
     }
+
+    private function mockRoleVoter(): RoleVoter
```

Comment:
```suggestion
    private function getRoleVoter(): RoleVoter
```

The method should be called `getRoleVoter()` because it does not return a mock object.

Reply:
Fixed in cf1ab5717.